examples/stackdriver-trace/README.md

```diff
@@ -0,0 +1,33 @@
+# Overview
+
+This example shows how to use [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) to instrument a simple Node.js application - e.g. a batch job - and export spans either to [Stackdriver Trace](https://cloud.google.com/trace/).
+
+## Installation
+
+```sh
+$ # from this directory
+$ npm install
+```
+
+## Authenticate
+
+You will need to create a service account and save the service account key json in the root of this example named `service_account_key.json`. For more information, visit <https://cloud.google.com/docs/authentication/>.
```

Comment:
Could you update the language here to explain that service accounts aren't needed of the user is running on GCP? I believe it says that in the main exporter README, so maybe bring similar language here?